utils.js

```diff
@@ -202,6 +203,9 @@ export async function postDailyLeetcodeMessage() {
   const weeklyRemainingDaysMessage = weeklyRemainingDays + (weeklyRemainingDays >= 2 ? " days" : " day");
 
   console.log(lastWeeklyProblemData);
+  
+  // TODO: refactor and call weekly update which includes leetcode and stats updates 
+  await new Stats(discordClient).postWeeklyStats();
```

Comment:
I'll do this refactor after this PR.

We can basically have a single process for handling our daily updates and our weekly updates:
```javascript
async sendDailyUpdate() {
   await leetcode.sendDaily();
   await stats.sendDaily();
}

async sendWeeklyUpdate() {
   await leetcode.sendWeekly();
   await stats.sendWeekly();
}

// some code for the cron job to send daily and weekly